sdk/healthinsights/azure-healthinsights-radiologyinsights/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Release History
+
+## 1.0.0b1 (2024-02-23)
```

Comment:
If you're planning on releasing this tomorrow or the day after this date should be updated to match the actual release date

Reply:
ok will change accordingly